lib/api/api_connect.dart

```diff
@@ -8,17 +9,27 @@ import 'package:Solon/api/comment.dart';
 import 'package:Solon/api/event.dart';
 import 'package:Solon/api/forumpost.dart';
 import 'package:Solon/api/vote.dart';
+import 'package:Solon/api/register.dart';
+import 'package:Solon/api/login.dart';
+import 'package:Solon/api/encryptPassword.dart';
 import 'dart:io';
 import 'package:flutter/services.dart';
 import 'package:intl/intl.dart';
+// import "package:pointycastle/export.dart";
+// import 'package:flutter_string_encryption/flutter_string_encryption.dart';
```

Comment:
please comment out imports that are not being used. clutters up the problems tab from actual problems that we have to deal with